lib/formatter/csv/classification.rb

```diff
@@ -51,6 +51,10 @@ def subject_data
         end.to_json
       end
 
+      def subject_ids
+          classification.subject_ids.map {|id| cache.subject(id).id }.to_json
```

Comment:
Why go from subject_ids to subjects and back to their ids?

 Also I was wondering: if we just join the ids with something, say a semicolon (so instead of jsonifying), then in most cases (projects with only one subject per classification) this column will simply look like a number that's easy to work with in R or Excel. What do you think of that?